jsonschema_extractor/attrs_extractor.py

```diff
@@ -37,30 +38,47 @@ def extract(cls, extractor, typ):
     def _extract_attribute(cls, extractor, attribute):
         is_required = attribute.default is attr.NOTHING
 
-        schema = None
         if "jsonschema" in attribute.metadata:
             schema = attribute.metadata["jsonschema"]
-        elif attribute.type is not None:
-            schema = extractor.extract(attribute.type)
         else:
+            schema = cls._extract_attribute_schema_by_type(extractor, attribute) or {}
             for validator in _iterate_validator(attribute.validator):
-                if isinstance(validator, _InstanceOfValidator):
-                    schema = extractor.extract(validator.type)
+                if isinstance(validator, _InValidator):
+                    schema['enum'] = validator.options
```

Comment:
It looks like there are cases where this could be a tuple, or an [enum.Enum](https://www.attrs.org/en/stable/api.html?highlight=in_#attr.validators.in_).

For completeness, should there be some handling of Enum values? I suppose would have to extract the enum value as option.